multiqc/modules/bases2fastq/plot_runs.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+
+from multiqc.plots import bargraph, linegraph, scatter, table
+
+"""
+Functions for plotting per run information of bases2fastq
+"""
+
+
+def plot_run_stats(runData, colorDict):
+    """
+    plot bargraph for polony numbers, Q30/Q40, index assignment rate and yields for each run
+    """
+    runNames = list(runData.keys())
+    runNames.sort()
+    NumPolonies = dict()
+    PercentQs = dict()
+    PercentAssigned = dict()
+    Yields = dict()
+    for run in runNames:
+        NumPolonies.update({run: {"Number of Polonies": runData[run]["NumPolonies"]}})
+        PercentQs.update({run: {"percent Q30": runData[run]["PercentQ30"], "percent Q40": runData[run]["PercentQ40"]}})
+        PercentAssigned.update({run: {"Successful assigned reads": runData[run]["PercentAssignedReads"]}})
+        Yields.update({run: {"Assigned yield": runData[run]["AssignedYield"]}})
+    plotContent = [NumPolonies, PercentQs, PercentAssigned, Yields]
+    config = {
+        "data_labels": [
+            {"name": "Polony Numbers", "ylab": "Number of polonies", "format": "{d}"},
+            {"name": "Base Qualities", "ylab": "Percentage"},
+            {"name": "Percentage assigned", "ylab": "Percentage"},
+            {"name": "Data Yield", "ylab": "Gb"},
+        ],
+        "cpswitch": False,
+        "stacking": None,
+        "description": "Bar plots to compare some general statistics of all found sequencing runs.",
+        "id": "run_stats_bar",
+        "title": "bases2fastq: General Sequencing Run QC stats plot",
+        "ylab": "QC",
+    }
+    plotName = "Sequencing Run QC metrics"
+    plotHtml = bargraph.plot(plotContent, pconfig=config)
+    anchor = "run_qc_metrics_plot"
+    description = "bar plots of general QC metrics"
+    helptext = """
+    This section shows and compare some metrics that indicate the quality of each sequencing run. 
+    
+    Polony numbers: total number of polonies. Each polony will yield one read1 and one read2
+    Base Qualities: percentage of bases that has >=30 (blue) or >=40 (black) base qualities
+    Percentage assinged: percentage of reads that has been assigned to any sample
+    Data Yield: the total volume of data that has been assigned to any sample
+    """
+    return plotHtml, plotName, anchor, description, helptext, plotContent
+
+
+def tabulate_run_stats(runData, colorDict):
+    """
+    Tabulate general information and statistics of each run
+    """
+    plotContent = dict()
+    for s_name in runData.keys():
+        runStats = dict()
+        runStats.update({"#Polonies": runData[s_name]["NumPolonies"]})
```

Comment:
`#Polonies` should be rounded to [read or bases counts](https://github.com/ewels/MultiQC/blob/41c1c85813c27a6cec2158d664594afacf178417/multiqc/utils/config_defaults.yaml#L29-L37C28) in the general stats table. See how [it's done](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/bcl2fastq/bcl2fastq.py#L394-L401
) for the bcl2fastq module as a reference.

Reply:
Same for other metrics. Please take a look if similar metrics that are already reported in other modules, and try to use the same naming style, value formatting, and color scheme. E.g. [bcl2fastq has](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/bcl2fastq/bcl2fastq.py#L462C1-L469) Q30, yield, mean base quality. FastQC can be a reference for other metrics and plots.